test/promql/tests/test_degraded_argoCD.yaml

```diff
@@ -0,0 +1,27 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+
+      # no argoCD app is degraded
+      - series: 'argocd_app_info{health_status="Degraded", dest_namespace="not-degraded"}'
+        values: '0x15'
+
+      # argoCD has a degraded app
+      - series: 'argocd_app_info{health_status="Degraded", dest_namespace="has-degraded"}'
+        values: '1x15'
+    alert_rule_test:
+      - eval_time: 15m
+        alertname: DegradedArgoCDApp
+        exp_alerts:
+          - exp_labels:
+              severity: Critical
+              dest_namespace: has-degraded
+              health_status: Degraded
+            exp_annotations:
+              summary: "ArgoCD service at has-degraded namespace includes an app which was 'Degraded'"
+              description: "ArgoCD service at has-degraded namespace includes an app which was 'Degraded' in the last 5 minutes"
```

Comment:
where do we see the last-5-min in the test?

Reply:
I added it just now, it is on lines 18-25.

Reply:
Is it ok now?

Reply:
LGTM